src/main/java/ru/skypro/homework/dto/Role.java

```diff
@@ -1,5 +1,16 @@
 package ru.skypro.homework.dto;
 
-public enum Role {
-    USER, ADMIN
+import org.springframework.security.core.GrantedAuthority;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+
+@Schema(description = "Роль пользователя")
+public enum Role implements GrantedAuthority {
+
+    ADMIN, USER;
+    @Override
+    public String getAuthority() {
+        return name();
```

Comment:
```suggestion
        return "ROLE_" + name();
```
вот надо сделать, чтобы всё работало, в прошлый раз писал